frontend/src/components/VSafeBrowsing/VSafeBrowsing.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <section id="safe-browsing" aria-labelledby="safe-browsing-heading">
+    <header class="relative mb-6 flex items-center justify-between">
+      <h4 id="safe-browsing-headers" class="caption-bold uppercase">
+        {{ $t("filters.safeBrowsing.title") }}
+      </h4>
+    </header>
+    <i18n tag="p" path="filters.safeBrowsing.desc" class="label-regular">
+      <template #sensitive>
+        <VLink :href="sensitivityPath">{{
+          $t("filters.safeBrowsing.sensitive")
+        }}</VLink>
+      </template>
+    </i18n>
+    <form class="safe-browsing-form">
```

Comment:
This is more of a pedagogical question, but why do we have something like `VLink` for links, presumably, but not something like `VForm` for more generic forms? Is it just because we have common functionality we want for links, but not for forms?

Reply:
> Is it just because we have common functionality we want for links, but not for forms?

Basically, yes. We have very few forms on the site so haven't needed a reusable abstraction. Also `form` elements are a non-visual wrapper for user inputs and don't really have any reusable functionality we need to encapsulate in our use case. This form, for example, only has a css class applied, so switching to `<VForm class="safe-browsing-form">` if it existed wouldn't offer any benefit.

Reply:
Ahh, I see! Thank you!